game_eggs/steamcmd_servers/v_rising/egg-v-rising.json

```diff
@@ -0,0 +1,231 @@
+            "container": "ghcr.io\/pterodactyl\/installers:debian",
+            "entrypoint": "\/bin\/bash"
+        }
+    },
+    "variables": [
+        {
+            "name": "[REQUIRED] Server Query Port",
+            "description": "UDP port for Steam server list features.",
+            "env_variable": "QUERY_PORT",
+            "default_value": "9877",
+            "user_viewable": false,
+            "user_editable": false,
+            "rules": "required|integer|between:1024,65536"
+        },
+        {
+            "name": "Automatic Updates",
+            "description": "Quickly checks for any server updates on startup, and updates if necessary.",
+            "env_variable": "AUTO_UPDATE",
+            "default_value": "1",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "boolean"
+        },
+        {
+            "name": "Game Mode",
+            "description": "What preset of game settings the server should run. Some settings may not apply after the save file is first created. Leave empty\/null if you are using a single-player uploaded save or a custom ServerGameSettings.json file.",
+            "env_variable": "GAME_SETTINGS_PRESET",
+            "default_value": "StandardPvP",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "nullable|string|in:,DuoPvP,HardcorePvP,Level30PvE,Level30PvP,Level50PvE,Level50PvP,Level70PvE,Level70PvP,SoloPvP,StandardPvE_Easy,StandardPvE_Hard,StandardPvE,StandardPvP_Easy,StandardPvP_Hard,StandardPVP"
```

Comment:
Game mode is only PvP or PvE
"GameModeType": "PvE",

Looks like you mixed up the variables

Reply:
This is not true. Please read the [documentation](https://github.com/StunlockStudios/vrising-dedicated-server-instructions#configuring-the-server). Current valid values are listed after the `in:` rule.

Reply:
You misunderstand me;

GameMode and GameModePresents are two different configuration settings.

So the name of the variable is wrong

Reply:
[Game-Server-Settings.pdf](https://github.com/parkervcp/eggs/files/8790745/Game-Server-Settings.pdf)

This is better for GameSettings :)

Reply:
The startup parameter named "Game Mode" sets the `GameSettingsPreset` setting in ServerHostSettings.json, **not** `GameMode` in ServerGameSettings.json. It was named "Game Mode" to be more recognizable than "Game Settings Preset" to the layman, but if just the name itself causes confusion I can change it. Advanced users are welcome to null this setting and use a custom ServerGameSettings.json file if they would like using the document you linked, but you already know this because the README states all of this.

Reply:
Default is:
`"GameSettingsPreset": "",`

Reply:
True, but assuming a `ServerGameSettings.json` override file is not present, a `null` value will default to the "default" `ServerGameSettings.json` file, which is a copy of `StandardPvP.json`, so it's effectively the same. We can change the default value for this setting to `null` if you would like, but it really doesn't make a difference.

Reply:
i'll do some tests atm:

for now:
RCON "true"; -> works
Secure "true" -> works

Reply:
I used the same rule before the last patch and it definitely didn't work. There is also a conversation on their Discord somewhere, because I had asked why it does not work with "VALUE", but only with VALUE.

It seems as if they have fixed this with the last patch quietly :)

"ListOnMasterServer": "true", -> works

Reply:
Commit [bd77d26](https://github.com/parkervcp/eggs/pull/1680/commits/bd77d26939a8b18887771d0d52fb0d4164996c93) should resolve this issue.